datafusion/sqllogictest/test_files/array.slt

```diff
@@ -2815,6 +2815,17 @@ SELECT  list_intersect(make_array(1,2,3), make_array(2,3,4)),
 ----
 [2, 3] [] [aa, cc] [true] [2.2, 3.3] [[2, 2], [3, 3]]
 
+query ??????
+SELECT  [1,2,3] && [2,3,4],
+        [1,3,5] && [2,4,6],
+        ['aa','bb','cc'] && ['cc','aa','dd'],
+        [true, false] && [true],
+        [1.1, 2.2, 3.3] && [2.2, 3.3, 4.4],
+        [[1, 1], [2, 2], [3, 3]] && [[2, 2], [3, 3], [4, 4]]
```

Comment:
As it is an alias, it is better to add `array_intersect` alongside to compare its value.

```suggestion
SELECT  [1,2,3] && [2,3,4], array_intersect([1,2,3], [2,3,4]),
        [1,3,5] && [2,4,6], array_intersect([1,3,5], [2,4,6]),
        ['aa','bb','cc'] && ['cc','aa','dd'], array_intersect(['aa','bb','cc'], ['cc','aa','dd']),
        [true, false] && [true], array_intersect([true, false], [true]),
        [1.1, 2.2, 3.3] && [2.2, 3.3, 4.4], array_intersect([1.1, 2.2, 3.3], [2.2, 3.3, 4.4]),
        [[1, 1], [2, 2], [3, 3]] && [[2, 2], [3, 3], [4, 4]], array_intersect([[1, 1], [2, 2], [3, 3]], [[2, 2], [3, 3], [4, 4]],),
```

Reply:
thank you! test has been fixed!